# Goal Request /moving_turtle/cmd_vel
float32 x
float32 y

---
# Result - time
uint64 duration
 
---
# Feedback /moving_turtle/pose
float32 x_pos 
float32 y_pos
float32 theta_pos